### PR TITLE
Feat : Add Navigation Component

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Nav-bar
Added Navigation bar.

### CODE REVIEW

File: assets/Css/ComponentsStyles.css [hunk 1]
```diff
@@ -478,4 +478,29 @@ main {
...
+  flex-wrap:wrap;
+  padding:0.5rem;
+  text-align: center;
+}
+.nav-bar-component__logo{
+  margin-left: 1rem;
+}
+.nav-bar-component__item{
+  display: inline-block;
+  margin-right: 0.5rem;
```
Comment (ShreyashMahajan) [inline assets/Css/ComponentsStyles.css:501]: use px for less than 1rem

**Review by ShreyashMahajan — COMMENTED** [review]
Approved

**Review by ShreyashMahajan — APPROVED** [review]
Approved

File: Components/Navigation.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+                    </div>
+                </div>
+                <div class="card-demo--container badge__code--container">
+                    <div class="code-snippet-container ">
+                        <textarea class="code-snippet" name="code-snippet" rows="4" cols="" readonly>&lt;div class=&quot;nav-bar-component__wrapper&quot;&gt;
+    &lt;div class=&quot;nav-bar-component--primary-actions&quot;&gt;
+        &lt;h1 class=&quot;nav-bar-component__logo&quot;&gt;Logo&lt;/h1&gt;
+    &lt;/div&gt;
+    &lt;div class=&quot;nav-bar-component--seconday-actions&quot;&gt;
+        &lt;ul class=&quot;nav-bar-component__list&quot;&gt;
```
Comment (mariyasada) [inline Components/Navigation.html:151]: for code, you can use carbon or thiscodeworks

**Review by mariyasada — APPROVED** [review]

File: Components/Navigation.html (hunk 2 above)
Comment (SaurabhChawda) [reply Components/Navigation.html:151]: Yes, I could use but I found that both are not responsive.

File: assets/Css/ComponentsStyles.css (hunk 1 above)
Comment (SaurabhChawda) [reply assets/Css/ComponentsStyles.css:501]: > use px for less than 1rem

Okay

**Pull request merged**